src/utils/agreements/getAgreements.js

```diff
@@ -0,0 +1,3 @@
+export default function getAgreements(agreement) {
+  return localStorage.getItem(agreement) === null ? 'false' : localStorage.getItem(agreement)
```

Comment:
Actually we never need string representation for this value, so it is better to write it as:
```
return localStorage.getItem(agreement) === 'true'
```